src/clib/pio_darray_int.c

```diff
@@ -96,42 +102,44 @@ int find_start_count(int ndims, int fndims, var_desc_t *vdesc,
 
     if (region)
     {
-        if (vdesc->record >= 0)
+        /* Allow extra outermost dimensions in the decomposition */
```

Comment:
I would like to understand this change better. Currently, if I have 4D data, with time as the first dimension, I do a 3D decomposition.

It seems that what you have done is allowed me to use all 4 dimensions in my decomposition, which would be less confusing for the user. Is that correct?

And you just want the extra outermost dim to be 1?